reference/opt/shaders-msl/asm/frag/combined-sampler-reuse.asm.frag

```diff
@@ -3,14 +3,14 @@
 
 using namespace metal;
 
-struct main0_in
+struct main0_out
```

Comment:
A lot of changes to these reference files, any reason for that?

Reply:
Yes. It's deliberate. No technical requirement...but it reads better.

Previous order of emitted structs was:

     in / out / uniforms

MSL functions read as:

    out funcName(in, uniforms)

and this new ordering of emitted matches that:

     out / in / uniforms

First the output then the inputs (in & uniforms).

Reply:
Ok, makes sense to me.